guides/azure-storage.md

```diff
@@ -6,37 +6,58 @@ permalink: /guides/azure-storage/
 
 1. Sign up for an [Azure Account](http://www.microsoft.com/windowsazure/offers/)
 2. Get your account and key
-	* Login to the [Windows Azure Portal](http://windows.azure.com)
-	* Click New Project, and then select Storage Account.
-	* Enter a project label and click Next.
-	* Enter the same name for the Storage Account Name. Make sure the account is available and then click _Create_
-		* This is your account in jclouds.
-	* Under the main azure screen, you should see Cloud Storage and the service you setup; Click on that.
+	* Login to the [Microsoft Azure Portal] (https://manage.windowsazure.com). The new portal (https://portal.azure.com) is in preview stage and hence instructions are not aligned with it.
+	* Click 'New', 'Data Servies', 'Storage' and 'Quick Create'.
+	* Enter the Storage Account Name. Make sure the account is available and then click _Create_
+		* This is your Storage account name you use in jclouds.
+	* Under the main Azure screen, you should see Cloud Storage and the service you setup; Click on that.
+	* Click on 'Manage Access Keys' at bottom of the page and copy Primary Access Key.
 		* Primary Access Key is the key you use in jclouds
 3. Ensure you are using a recent JDK 6
 4. Setup your project to include `azureblob`
 	* Get the dependency `org.apache.jclouds.provider/azureblob` using jclouds [Installation](/start/install).
-5. Start coding
+5. Setup your project to include 'guava'. Include following dependency to jclouds [Installation](/start/install). Update the version mentioned in dependency below to the latest available version.
+	<dependency>
+		<groupId>com.google.guava</groupId>
+		<artifactId>guava</artifactId>
+		<version>18.0</version>
+	</dependency>
```

Comment:
Ok. My assumption was that you would typically use latest version to avoid problems.